### PR TITLE
Chores/upgrade to elixir 1.6

### LINKED ISSUE #25: Cache CircleCI 2 build dependencies
These dependency installs should be cached:
- PhantomJS
- npm & node
- yarn
- brunch

### PR DESCRIPTION
**Pull request by craiglyons**
* Upgrade to elixir 1.6
* Simplify CI
* Simplify brunch steps
* Apply formatter

closes #25

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,40 +2,15 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/elixir:1.5.1
+      - image: circleci/elixir:1.6-node-browsers
```
Comment (outofambit) [inline .circleci/config.yml]: does this image effectively cover the `run` steps removed below?

**Review by outofambit — APPROVED** [review]
this all seems very reasonable! just one non-blocking question

File: .circleci/config.yml (hunk 1 above)
Comment (craiglyons) [reply .circleci/config.yml]: That's exactly right, here's the doc:
https://circleci.com/docs/2.0/circleci-images/#language-image-variants

Comment (outofambit) [reply .circleci/config.yml]: wow, super cool!

**Pull request merged**